docs/docs.md

```diff
@@ -201,6 +201,31 @@ type Data = {
 
 # `/event/{eventKey}/matches`
 
+## `PUT`
+
+### Request
+
+```ts
+type Data = {
+  redAlliance: string[]
+  blueAlliance: string[]
+  // UTC date - match predicted time
```

Comment:
Can you make this comment say scheduled time? You can override the property on the object using a union, even if the value is the same and it will update the comment